src/repositories/request-repository.ts

```diff
@@ -83,7 +86,8 @@ const findRequestsToAnchor = (connection: Knex, now: Date): Knex.QueryBuilder =>
       .orWhere((subBuilder) =>
         subBuilder
           .where({ status: RequestStatus.FAILED })
-          .andWhere('createdAt', '>=', earliestDateToRetryFailed)
+          .andWhere('createdAt', '>=', earliestFailedCreatedAtToRetry)
+          .andWhere('updatedAt', '<=', earliestFailedUpdatedAtToRetry)
```

Comment:
is this really `latestFailedUpdatedAtToRetry` since we're looking for requests with updatedAt older than this?

Reply:
yes 👍

Reply:
good catch